lib/chef/application/windows_service.rb

```diff
@@ -200,8 +200,8 @@ def run_chef_client
           timeout: Chef::Config[:windows_service][:watchdog_timeout],
           logger: Chef::Log
         )
-        Chef::Log.trace "#{result.stdout}"
```

Comment:
I guess it's more clear what we're trying to do here, but this seems a bit odd

Reply:
the parens are maybe unnecessary but `result.stdout.to_s` is definitely better than `"#{result.stdout}"`